release-notes/3.10.10.md

```diff
@@ -0,0 +1,30 @@
+# Release notes for Gluster 3.10.10
+
+This is a bugfix release. The release notes for [3.10.0](3.10.0.md), [3.10.1](3.10.1.md), [3.10.2](3.10.2.md), [3.10.3](3.10.3.md),
+[3.10.4](3.10.4.md), [3.10.5](3.10.5.md), [3.10.6](3.10.6.md), [3.10.7](3.10.7.md), [3.10.8](3.10.8.md) and [3.10.9](3.10.9.md) contain a listing of all
+the new features that were added and bugs fixed in the GlusterFS
+3.10 stable release.
+
+## Major changes, features and limitations addressed in this release
+**No Major changes**
+
+## Major issues
+
+1. FIXED: Expanding a gluster volume that is sharded may cause file corruption
```

Comment:
Compared to this section in previous release notes, this change can be a little ambiguous to users. Is the issue completely fixed or fixed but not completely i.e be cautious while using ?

Reply:
Ok couple of things here,
- The release notes for 3.13.2 also reads the same hence may need correction
- The statement below is ambiguous "... is still fixed..." should have read something like "... is fixed ..."

Overall before making the changes wanted to understand the comment better,
- So the first bullet (1) is ambiguous (noted)
- Ambiguity arises due to just pre-pending the FIXED to older content?
  - I assume your comment is on this, which means we need to highlight this differently rather than just pre-pending a FIXED keyword, would that be right?
- Ambiguity exists in the statements below as noted and will be corrected
- Anything else?

Reply:
I intended to mean that the pre-pending and the following statement can possibly cause confusion to users:

> The last known cause for corruption (#1498081) is still fixed with this release.

Also, the `Major issues` section, IIRC, in previous releases pointed at issues that were not fixed - it was a notice of caution to users. Now, the first issue under the same section is an announcement that an old issue has been fixed (partially? IDK) and the second issue is an issue that still exists. May be highlight point 1 under `Bugs addressed` or `Major issues fixed` ?